plugin_repackaging.sh

```
@@ -12,183 +12,155 @@ PIP_MIRROR_URL="${PIP_MIRROR_URL:-$DEFAULT_PIP_MIRROR_URL}"
-	echo "Repackage success."
+    local PKG_PATH=$1
+    local PKG_FULL=$(basename "$PKG_PATH")
+    local PKG_NAME="${PKG_FULL%.*}"
+
+    echo "Unziping ..."
+    install_unzip
+    unzip -o "$PKG_PATH" -d "${CURR_DIR}/${PKG_NAME}"
+    [[ $? -ne 0 ]] && echo "Unzip failed." && exit 1
+
+    echo "Repackaging ..."
+    cd "${CURR_DIR}/${PKG_NAME}"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)